public/data/RajPakhurde.json

```
@@ -0,0 +1,33 @@
+{
+    "name": "Raj Pakhurde",
+    "type": "personal",
+    "bio": "Open Source Enthusiast!",
+    "avatar": "https://github.com/RajPakhurde.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/RajPakhurde",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/raj_pakhurde",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Linkdein",
+        "url": "https://linkedin.com/in/raj-pakhurde-78b297228",
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Front-end web-development",
+        "date": "December 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started learning web-development",
+        "url": "https://showwcase.com/rajpakhurde"
+      }
+    ]
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```